### PR TITLE
[eclipse/xtext#1948] create r202103 target platform

### PR DESCRIPTION
**Pull request by cdietrich**
[eclipse/xtext#1948] create r202103 target platform
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: releng/org.eclipse.xtend.target/org.eclipse.xtend.target-r202103.target [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+		<unit id="org.eclipse.draw2d.feature.group" version="0.0.0"/>
+		<unit id="org.eclipse.emf.mwe2.lib" version="0.0.0"/>
+		<unit id="org.eclipse.emf.mwe2.launcher.feature.group" version="0.0.0"/>
+		<unit id="org.eclipse.m2e.feature.feature.group" version="0.0.0"/>
+		<repository location="https://download.eclipse.org/releases/2021-03"/>
+	</location>
+
+	<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+		<unit id="org.eclipse.emf.sdk.feature.group" version="0.0.0"/>
+		<repository location="https://download.eclipse.org/modeling/emf/emf/builds/release/2.25"/>
```
Comment (kthoms) [inline releng/org.eclipse.xtend.target/org.eclipse.xtend.target-r202103.target:99]: https://download.eclipse.org/modeling/emf/emf/builds/release/2.24

Comment (cdietrich) [reply releng/org.eclipse.xtend.target/org.eclipse.xtend.target-r202103.target:99]: 2.25 should be released soon

**Comment by cdietrich** [comment]
@kthoms can you please re-review?

File: releng/org.eclipse.xtend.target/org.eclipse.xtend.target-r202103.target (hunk 1 above)
Comment (kthoms) [reply releng/org.eclipse.xtend.target/org.eclipse.xtend.target-r202103.target:99]: fine now

**Review by kthoms — APPROVED** [review]

**Pull request merged**